spec/interactors/recipes/create_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+RSpec.describe Recipes::Create do
+  describe ".call" do
+    include_context :interactor
+
+    let(:initial_context) do
+      {
+        recipe: recipe,
+        author: user
+      }
+    end
+
+    let(:user) { create :user }
+    let(:recipe) { build :recipe, recipe_params }
```

Comment:
`let(:recipe) { build, title: title, content: "Content" }` and we can remove `recipe_params` after that